lib/srv/regular/sshserver.go

```diff
@@ -1220,10 +1220,12 @@ func (s *Server) listenTCPIP(scx *srv.ServerContext, addr string) (net.Listener,
 
 func controlSyscallConn(conn syscall.RawConn, f func(fd uintptr) error) error {
```

Comment:
Considering that `(syscall.Conn).SyscallConn()` is never really supposed to fail, perhaps it would be convenient to have this helper take a `syscall.Conn` and do the conversion manually.